### DISCUSSION TITLE
Fooocus MRE

### DISCUSSION THREAD

**Discussion by drdancm** [LykosAI/StabilityMatrix#151] [Ideas]
Unless I misunderstand the MRE version has more advanced features such as ControlNET.

https://github.com/MoonRide303/Fooocus-MRE

Would it be possible to add this as an additional package?

Thanks,

Dan

**Comment by mohnjiles** [comment]
Hi, this was added in [v2.4.0](https://github.com/LykosAI/StabilityMatrix/releases/tag/v2.4.0) - you should be able to install it from the Packages tab on v2.4 or later.

**Comment by ScionoicS** [comment]
I went to install this version through Matrix today, and it says it's never updated so it may be removed. I think the default branch that matrix pulls from is the stable release, while dailies are on /main branch.  It's updated often and is still on the cutting edge of base fooocus.  I'm trying to consolidate all the UI's i try into the matrix folder, and i'd love if this one stays as its one i use.
